### ISSUE TITLE
The loading animation on Indicator page doesn't have an alt tag

### ISSUE THREAD

**Issue by SavvasStephanides** [open-sdg/open-sdg#738]
**Please complete the following information** 
Site repository URL:  github.com/open-sdg/open-sdg
Open SDG version: 1.0.0

**Describe the bug**

Running an accessibility test with Pa11y returns this error against indicator pages with sections on top of the charts ([example](http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/opensdg1/3-4-2/)). 

**To Reproduce**

1. Grab the URL of an indicator with a "catch-all" section on top
2. Run a Pa11y accessibility test
3. Pa11y will alert that the loading animation doesn't have an `alt` tag (see screenshot)

**Expected behavior**

All images should have an alt tag for accessibility.

**Screenshots**

![image](https://user-images.githubusercontent.com/478770/84400474-fd1d8c80-abf9-11ea-8d36-050ebad07ba8.png)

**Desktop (please complete the following information):**

Tested on desktop configuration, including high contrast

**Smartphone (please complete the following information):**

Also tested on mobile configuration of 320x480

**Comment by brockfanning** [comment]
Should be fixed by #758
